### PR TITLE
Feature/dsnd 783 company links consumer deletes integration test stream charges

### PR DESCRIPTION
**Pull request by ipg-ch**

### CODE REVIEW

File: src/main/java/uk/gov/companieshouse/company/links/service/ChargesService.java [hunk 1]
```diff
@@ -43,10 +43,18 @@ public ApiResponse<ChargesApi> getCharges(String contextId, String companyNumber
...
-                .getCharges(uri));
+        ApiResponse<ChargesApi> result = null;
+        try {
+            result = executeOp(contextId, "getCharges", uri,
+                getApiClient(contextId)
+                    .privateDeltaChargeResourceHandler()
+                    .getCharges(uri));
+        } catch (Exception ex) {
+            String message = ex.getMessage();
+            logger.info(message);
```
Comment (zahidm3) [inline src/main/java/uk/gov/companieshouse/company/links/service/ChargesService.java:54]: Exception logged as info, do we need to this new try/catch?

File: src/test/java/uk/gov/companieshouse/company/links/processor/InsolvencyStreamProcessorTest.java [hunk 2]
```diff
@@ -63,21 +63,6 @@ void successfullyProcessResourceChangedDataInsolvencyLinksGetsUpdated() throws I
 
 
         insolvencyProcessor.processDelta(mockResourceChangedMessage);
-
-        verify(companyProfileService).getCompanyProfile(CONTEXT_ID, MOCK_COMPANY_NUMBER);
-        verify(logger, times(2)).trace(anyString());
-        verify(logger, atLeastOnce()).trace(
-                contains("Resource changed message with contextId context_id of kind company-insolvency"));
-        verify(logger, atLeastOnce()).trace(String.format(
```
Comment (zahidm3) [inline src/test/java/uk/gov/companieshouse/company/links/processor/InsolvencyStreamProcessorTest.java:71]: Not sure why we removing all these assertions?

**Review by zahidm3 — CHANGES_REQUESTED** [review]

File: src/test/java/uk/gov/companieshouse/company/links/processor/InsolvencyStreamProcessorTest.java (hunk 2 above)
Comment (ipg-ch) [reply src/test/java/uk/gov/companieshouse/company/links/processor/InsolvencyStreamProcessorTest.java:71]: They are pointless only asserting the log messages at trace level which are not a requirement.

Comment (zahidm3) [reply src/test/java/uk/gov/companieshouse/company/links/processor/InsolvencyStreamProcessorTest.java:71]: Agree but by mistake you removed valid assertion as well.

verify(companyProfileService).getCompanyProfile(CONTEXT_ID, MOCK_COMPANY_NUMBER);

**Review by zahidm3 — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/companieshouse/company/links/service/ChargesService.java (hunk 1 above)
Comment (ipg-ch) [reply src/main/java/uk/gov/companieshouse/company/links/service/ChargesService.java]: Was me trying to find out why integration test was behaving unexpectedly. Should have been removed, blame being tired on forgetting that!

**Review by zahidm3 — APPROVED** [review]

**Pull request merged**